server/pom.xml

```diff
@@ -81,6 +81,11 @@
             <version>${spring.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+            <version>5.2.1</version>
+        </dependency>
```

Comment:
Underlying library for Spring's `TestRestTemplate` has been changed (see migration guide). We need to import it explicitly in order for our integration tests to work. Otherwise, the JVM runtime falls back to the standard JDK libraries (which is bad because we don't get compile time errors, but mysterious test fails).